src/Saas.SignupAdministration/Saas.SignupAdministration.Web/Program.cs

```diff
@@ -65,7 +65,10 @@
 
 // builder.Configuration to sign-in users with Azure AD B2C
 builder.Services.AddMicrosoftIdentityWebAppAuthentication(builder.Configuration, Constants.AzureAdB2C)
```

Comment:
Certainly cleaner than before, my only concern is that all of our other examples use the full URI, might be a little confusing.

Reply:
I explained this in the project readme already (on the docs-updates branch). I'll clean up our other examples too.